.github/workflows/scheduled.yml

```
@@ -0,0 +1,136 @@
+name: Nightly tests
+
+on:
+  schedule:
+    - cron: '10 * * * *'
```

Is this not hourly? For nightly I think we want to run at something like 0400Z (being UK-centric as I am):
```suggestion
    - cron: '* 4 * * *'
```

---

This is me setting up frequent running while attempting to get the timer to initially run on my personal fork. I'll do anything further in another branch to avoid spamming the PR discussion.